app/src/main/java/com/gh4a/fragment/UserFragment.java

```diff
@@ -85,8 +86,8 @@ protected void onResultReady(User result) {
         @Override
         protected Loader<LoaderResult<Collection<Repository>>> onCreateLoader() {
             Map<String, String> filterData = new HashMap<>();
-            filterData.put("sort", "pushed");
-            filterData.put("affiliation", "owner,collaborator");
+            filterData.put(Constants.EXTRA_SORT, Constants.PUSHED);
```

Comment:
Probably move to [Repository.java](https://github.com/slapperwan/gh4a/blob/master/github-api/src/main/java/org/eclipse/egit/github/core/Repository.java)?

Reply:
I moved it to `IGithubConstants`